maven_iuvs/echelle.py

```diff
@@ -545,13 +708,15 @@ def coadd_lights(data, n_good):
            detector image data, size (frames, spatial, wavelength)
     n_good : int
              number of valid frames of data used, calculated by subtract_darks.
-
+    exclude_frames : list
```

Comment:
does not seem to be a parameter in definition above?

Reply:
also missing in function below... is the docstring change wrong?

Reply:
Hmmm yeah I think this is an artifact. I think it's not needed because I do nansum to ignore any bad values, and n_good should be calculated correctly. I've been testing various quicklooks and they all looked okay. I'll double check tomorrow but for now I've removed this in 2deddcd9215b5960a570053b65ca16ec583551a1